api/api.proto

```diff
@@ -3414,6 +3414,8 @@ message ClusterPairCreateRequest {
   string remote_cluster_token = 3;
   // Set the new pair as the default
   bool set_default = 4;
+  // If the pair is being created for DR mode
+  bool disaster_recovery_mode = 5;
```

Comment:
Suggestion:Should this be a enum rather than bool,  to allow any flavors of disaster recovery?

Reply:
Yeah, that's a good point. What should the types be though? Should the name be Mode, and different modes be Default and DisasterRecovery for now?

Reply:
Yeah that sounds great. default would be disaster_recovery_mode_off.

Reply:
Updated. Can you take a look again?

Reply:
gook great!.